app/models/application.rb

```diff
@@ -0,0 +1,11 @@
+class Application < ApplicationRecord
+  has_many :application_pets
+  has_many :pets, through: :application_pets
+  validates :name, presence: true
+  validates :street_address, presence: true
+  validates :city, presence: true
+  validates :state, presence: true
+  validates :zip_code, numericality: true
+  validates :description, presence: true
+end
```

Comment:
Put some separation between your associations and validations to make this code easier to read.